README.md

```diff
@@ -110,9 +110,10 @@ memorized or referenced in order to run said task.
 Consider formatting as an example. The default approach to formatting files is
 to run `dartfmt -w .`. But, some projects may want to exclude certain files that
 would otherwise be formatted by this command. Or, some projects may want to use
-`pub run dart_style:format` instead of `dartfmt`. Currently, there is no
-project-level configuration supported by the formatter, so these sorts of things
-just have to be documented in a `README.md` or `CONTRIBUTING.md`.
+`dart run dart_style:format` instead of `dartfmt` or `dart format` instead of 
+`dartfmt`. Currently, there is no project-level configuration supported by the 
```

Comment:
Does this line make sense?

Reply:
Hmm.. since the purpose of this section is to explain why dart_dev itself is useful, I think we can leave it how it was except with `dartfmt` swapped for `dart format`, since the latter is now the canonical SDK tool.